client/src/components/BestSellerSection.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ProductCard from './Card/ProductCard';
+import '../home.css';
+
+const BestSellerSection = () => {
+  return (
+    <div className='best-seller-section'>
+        <h2 className='title-section'>Best Seller</h2>
+      <div className="seller">
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
+        <ProductCard />
```

Comment:
What is this ?

Reply:
this is a fake data i put it to check layout for entire section